src/_functions.scss

```diff
@@ -12,14 +12,14 @@
   @return ((100 / $columns) * $position) * $unit;
 }
 
-/// Verifies if the value is a percentage or another kind of length in order
-/// to remove or return a percentage value to be used as a multiplier unit
+/// Gets the multiplier unit based on a value.
```

Comment:
Use periods at the end of comment sentences everywhere or nowhere, for consistency.